.github/ISSUE_TEMPLATE/api_bug.md

```diff
@@ -0,0 +1,117 @@
+---
+name: API Bug
+about: "Report a bug you have encountered using SeAT API."
+---
+
+# Bug
+*This template contains an example bug report. Please replace all text except for the checklist and the section headers (they start with \#). Thank you for your contribution to improving SeAT.*
+
+*In this space, please provide a short description of the bug you've encountered.
+This should help us to understood what you've did while you get the error. E.g.:*
```

Comment:
it's a little informal: 
Maybe: `This should help us reproducing the error. E.g:`